example/scalalib/native/3-multi-module/build.mill

```diff
@@ -0,0 +1,91 @@
+//// SNIPPET:BUILD
+package build
+import mill._, scalalib._, scalanativelib._
+
+trait MyModule extends ScalaNativeModule {
+  def scalaVersion = "2.13.11"
+  def scalaNativeVersion = "0.5.5"
+
+  object test extends ScalaNativeTests {
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.4")
+    def testFramework = "utest.runner.Framework"
+  }
+
+  def nativeLinkingOptions = Seq("-L" + millSourcePath.toString + "/target")
+}
+
+object foo extends MyModule {
+  def moduleDeps = Seq(bar)
+
+  def ivyDeps = Agg(ivy"com.lihaoyi::mainargs::0.7.6")
+
+  // Compile C
+  def nativeCompiled = T {
+    os.makeDir.all(millSourcePath / "target")
+
+    os.proc(
+      "gcc", "-m64", "-shared", "-fPIC",
+      "-I", "/workspaces/milling-two/sunday-work/mill/example/scalalib/native/3-multi/bar/native-src",
```

Comment:
Hardcoded paths are wrong

Reply:
Yes, I know. I can now replace it with what you informed me of how to use paths from one module in another one